guide/docs/interactions/select-menus.mdx

```diff
@@ -1,8 +1,291 @@
+2.  Remove the view altogether leaving only the original message.
+
+For this example we will disable the component `on_timeout`. However, if you wish to remove the view completely you can pass `view=None` instead of `view=self` as the example below shows.
+
+We'll continue with the original example from above, however I will only be altering the parts that matter.
+
+```python title="viewtimeout.py"
+...
+...
+
+
+class DropDownView(disnake.ui.View):
+
+    message: disnake.Message  # adding to typehint the future `message` attribute
+
+    def __init__(self):
+        # this time we will set the timeout to 30.0s
+        super().__init__(timeout=30.0)
+        # now let's add the `StringSelect` object we created above to this view
+        self.add_item(AnimalDropdown())
+
+    # to handle this timeout, we'll need to override the default `on_timeout` method within the `View``
+    async def on_timeout(self):
+        # now we will edit the original command response so that it displays the disabled view
+        # since `self.children` returns a list of the components attached to this `View` and we want to
+        # disable the single component, we can access it with a simple [0] index
+        self.children[0].disabled = True
+        # now we edit the message with this updated `View`
+        await self.message.edit(view=self)
+
+
+...
+...
+# the only changes we need to make to handle the updated view is to fetch the original response after
+# it has been sent.  This is required because `interaction.send` methods do not return a message object
+@bot.slash_command()
+async def animals(inter: disnake.ApplicationCommandInteraction):
+    """Sends a message with our dropdown containing the animals"""
+
+    # create the view with our dropdown object
+    view = DropDownView()
+
+    # respond to the interaction with a message and our view
+    await inter.response.send_message("What is your favorite type of animal?", view=view)
+
+    # this will add a new `message` attribute to the `DropDownView` that will be edited
+    # once the view has timed out
+    view.message = await inter.original_response()
+
+
+...
+...
+```
+
+### Views vs. low-level components
+
+As an alternative to using `Views`, it is possible to use SelectMenus as low-level components.
+These components do not need to be sent as part of a View and can be sent as is. However, you should take note
+that any each component being sent in this manner must have a `custom_id` explicitly set. It should also be noted
+that components will share listeners. This means `custom_id` should be unique for each component.
+
+The main advantage of this is that listeners, by nature, are persistent and will remain fully functional over bot reloads. Listeners are are stored in the bot
+strictly once, and are shared by all components. Because of this, the memory footprint will generally be smaller
+than that of an equivalent view.
```

Comment:
```suggestion
As an alternative to using `View`s, it is possible to use Select Menus as low-level components.
These components do not need to be sent as part of a View and can be sent as is.

Note that any component being sent in this manner must have a `custom_id` explicitly set. Component interactions are sent to all listeners,
which means the `custom_id` should be unique for each component to be able to identify the component in your code.

The main advantage of this is that listeners, by nature, are persistent and will remain fully functional over bot reloads. Listeners are stored in the bot
strictly once, and are shared by all components. Because of this, the memory footprint will generally be smaller
than that of an equivalent view.
```

again, minor formatting changes and small details